packages/store-ui/src/atoms/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { forwardRef } from 'react'
+
+export interface SkeletonProps {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+}
+
+const Skeleton = forwardRef<HTMLDivElement, SkeletonProps>(function Skeleton(
+  { testId = 'store-skeleton', ...props },
+  ref
+) {
+  return (
+    <div ref={ref} data-store-skeleton-container data-testid={testId}>
+      <div data-store-skeleton {...props} />
+    </div>
```

Comment:
I like this approach, but what do you think about using only one div?

Reply:
Yeah, it's better! I'll update it